guide/english/c/math/index.md

```diff
@@ -31,7 +31,7 @@ In the above example, the variable `result` will be equal to a multiplied by b.
 #### Division: `/`
 Division is performed with a `/`, like so:
 ```C
-result = a / b;
+result = a / b;   // iff b is not equal to 0
```

Comment:
Please only use abbreviations that have already been explained in the article, this may be confusing to less experienced readers.

Reply:
This one has gone stale and the user has deleted branch

Reply:
@cmccormack I think this one can be closed, because we can not push back to a branch (it has been deleted).